exporters/tests/test_failure_exporter_pager_duty.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
```

Comment:
This is mostly integration test, not really unit ones, which could fake the data from the requests() and would not in such case require TOKEN to be provided.

Reply:
just saying we could avoid calling the real API for other type of the tests and the coverage of this exporter would be still met.

Reply:
I am not talking here about mockoon, just mock the requests, e.g.:

```
mock_session = Mock(spec=requests.Session)
mock_json_response = /get issues json file, even by using json.load()/
mock_response.json.return_value = mock_json_response
mock_session.get.return_value = mock_response
with patch('requests.Session', return_value=mock_session):
   // test your function
```